tribeiro/MTAOS/AddAberration.ipynb

```diff
@@ -0,0 +1,896 @@
+{
```

Comment:
<p>I would note the following here -</p><ul><li>Need w_2021_12 to run, w_13 (and probably higher versions) won't run.</li><li>need MTAOS, wep, ofc</li><li>best run on teststand, not lsp, because MTAOS needs to import dds library</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>